gsc/_t-univ-4.scm

```diff
@@ -1068,7 +1068,9 @@
      (return
       (^flonum-box
        (if arg2
-           (^float targ-inexact-+0) ;; TODO: implement 2 argument fllog
+           (univ-ieee/ ctx
```

Comment:
`_num.scm` has special code for when both arguments are positive or negative powers of 2.  I don't know whether it's important that the universal backend share that behavior.

Reply:
When the 2 argument `##fllog` is inlined by the C backend, it calls the following C macro

```
#define ___F64LOG2(x,y)(___CLIBEXT(log)(x)/___CLIBEXT(log)(y))
```

so the definition in _num.scm is not used (so the universal backend is doing the same here and just dividing the logs).